markdown/1.2/en/ff-middleware.md

```diff
@@ -125,3 +125,27 @@ window.addEventListener("ffReady", () => {
 ```
 
 See the [API tab](/api/1.2/ff-middleware#tab=api) for more details on configuration options.
+
```

Comment:
I wonder if we should replace the text from api/1.2/ff-product-teaser-campaign-processor through a link to the middleware? Or remove it from the sidebar and add a section to `Campaign` with only a link to the middleware here?

I am not too happy with having almost the exact same text and same feature description in two places written down separately

Reply:
Grouping all campaigns in some way would be useful I think. It wouldn't bloat the left navigation as much.
I'll create a separate issue for that. For now it just has to be _good enough_ for the next release.